kratos/input_output/base_64_encoded_output.h

```diff
@@ -37,7 +37,7 @@ namespace Kratos {
  *  @note The destructor of the instance will write the remaining bytes of the data container with padding.
  *        Hence, scoping of the encoder should be done to properly finish writing of the data stream
  */
-class Base64EncodedOutput
+class KRATOS_API(KRATOS_CORE) Base64EncodedOutput
```

Comment:
I'm not sure this should be exported. Isn't this only used in core sources?

Reply:
See [Move part of the code to source file](https://github.com/KratosMultiphysics/Kratos/pull/11139/commits/40cdb54d5b0585e95c83f4393049cee088623fa7)

Reply:
but that source file is still part of Core, right? If so, it shouldn't need `KRATOS_API`.

by the way, `Base64EncodedOutput::EncodeByteTriplet` is invoked very frequently, so it should probably stay inlined.

Reply:
Technically is not inlined, we should explicitly inline it and even that way compiler would ignore it.

Reply:
> but that source file is still part of Core, right? If so, it shouldn't need `KRATOS_API`.
 I don't trust windows